cmd/main.go

```diff
@@ -148,6 +149,17 @@ func initServices(_ context.Context, p common.Platform) error {
 	})
 }
 
+// registerWebhooks registers all resource webhooks with the manager.
+func registerWebhooks(mgr ctrl.Manager) error {
+	if err := datasciencecluster.RegisterWebhooks(mgr); err != nil {
+		return fmt.Errorf("unable to register DataScienceCluster webhooks: %w", err)
+	}
+	if err := dscinitialization.RegisterWebhooks(mgr); err != nil {
+		return fmt.Errorf("unable to register DSCInitialization webhooks: %w", err)
```

Comment:
Just a doubt here,
  Each time RegisterWebhooks is called with mgr, it registers a new webhook handler with the same webhook server. Is my understanding correct?